src/i18n/locales/ca/tools.json

```diff
@@ -1,13 +1,14 @@
 {
-	"readFile": {
-		"linesRange": " (línies {{start}}-{{end}})",
-		"linesFromToEnd": " (línies {{start}}-final)",
-		"linesFromStartTo": " (línies 1-{{end}})",
-		"definitionsOnly": " (només definicions)",
-		"maxLines": " (màxim {{max}} línies)"
-	},
-	"toolRepetitionLimitReached": "Roo sembla estar atrapat en un bucle, intentant la mateixa acció ({{toolName}}) repetidament. Això podria indicar un problema amb la seva estratègia actual. Considera reformular la tasca, proporcionar instruccions més específiques o guiar-lo cap a un enfocament diferent.",
-	"codebaseSearch": {
-		"approval": "Cercant '{{query}}' a la base de codi..."
-	}
+  "readFile": {
```

Comment:
Is it intentional that the indentation changed on all of these? Can we run prettier to make sure?